cypress/e2e/admin/administrateMigration.feature

```diff
@@ -18,9 +18,21 @@ Feature: Admin Users - To administrate the Migration process.
     When I click on agree migration button
     Then I see the migration is active field
     Then I see the end migration button
+    Then I see the migration mandatory switch is not checked
 
+  Scenario: Admin toggles ldap sync for school in migration
+    Then I see the sync during migration switch is visible and not checked
+    When I toggle the sync during migration switch
+    Then I see the sync during migration switch is checked
 
-  Scenario: Admin ends the migration
+  Scenario: Admin stops the migration
     When I click on end migration button
+    Then I see the end of migration information title
+    Then I see the end of migration information text
+    Then I see the end migration information link href is blog.niedersachsen.cloud/umzug
+    Then I see the end migration confirmation checkbox is unchecked
+    Then I see the abort button for end of migration conformation
+    Then I can see the end migration confirmation button is disabled
     When I click on the end migration confirmation checkbox
-    When I click on the end migration confirmation button
+    Then I see the end migration confirmation button is enabled
+    When I click on the end migration confirmation button
```

Comment:
What's about the migration timestamps which will be displayed when a migration was stopped?